uninstall.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Exit immediately if something fails
+set -e
+
+cd build
+sudo make uninstall
+kquitapp5 krunner 2> /dev/null; kstart5 --windowclass krunner krunner > /dev/null 2>&1 &
```

Comment:
```suggestion
kquitapp5 krunner 2> /dev/null; kstart5 --windowclass krunner krunner &> /dev/null &
```

Reply:
Actually we can simplify this without the pipes now, some time ago we got annoying warnings.